### PR TITLE
comment_user

### PR DESCRIPTION
**Pull request by zero060699**
## Related Tickets:
- [#33014](https://edu-redmine.sun-asterisk.vn/issues/33014)

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by zero060699** [comment]
ready

File: app/Http/Controllers/CommentController.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    public function store(Request $request)
+    {
+        $post = Post::find($request->post_id);
+        $data = [
+            'message' => "errors",
+        ];
+        if (empty($post)) {
+            return json_encode($data);
+        }
+        $check = comment::create([
```
Comment (huyenttm-0901) [inline app/Http/Controllers/CommentController.php]:
```suggestion
        $check = Comment::create([
```

**Review by minhkute10399 — APPROVED** [review]

**Comment by zero060699** [comment]
ready

File: app/Http/Controllers/CommentController.php [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    public function index(Post $post)
+    {
+        return response()->json($post->comments()->with('user')->latest()->get());
+    }
+
+    public function store(Request $request)
+    {
+        $post = Post::find($request->post_id);
+        $data = [
+            'message' => "errors",
```
Comment (huyenttm-0901) [inline app/Http/Controllers/CommentController.php]: i18n

File: app/Http/Controllers/CommentController.php [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+        if (empty($post)) {
+            return json_encode($data);
+        }
+        $check = Comment::create([
+            'content' => $request->content,
+            'post_id' => $request->post_id,
+            'user_id' => Auth::id(),
+        ]);
+        if ($check) {
+            $data['message'] = "success";
```
Comment (huyenttm-0901) [inline app/Http/Controllers/CommentController.php]: tương tự

**Comment by zero060699** [comment]
ready

**Review by huyenttm-0901 — APPROVED** [review]

**Pull request merged**